proxy/auth_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Stefan Prodan 2017. All rights reserved.
```

Comment:
Is this copied from Lewis PR or visa-versa?

https://github.com/openfaas/faas-cli/pull/180

If so is the Copyright notice appropriate?

Reply:
The hashing comparison it is copied. 

Regarding Copyright wouldn't be better to put "OpenFaaS Team" and not the contributor name? I've seen this approach on other projects. Each commit is signed, so if someone wants to see the actual contributors it can from the commit history.

Reply:
If you like you can put OpenFaaS authors on your new commits in this PR. Unlikely to retrofit across all repos at this point.